libp2p/crypto/crypto.nim

```diff
@@ -655,16 +709,18 @@ proc getOrder*(remotePubkey, localNonce: openarray[byte],
   ctx.update(localPubkey)
   ctx.update(remoteNonce)
   var digest2 = ctx.finish()
-  var mh1 = MultiHash.init(multiCodec("sha2-256"), digest1)
-  var mh2 = MultiHash.init(multiCodec("sha2-256"), digest2)
+  var mh1 = ? (MultiHash.init(multiCodec("sha2-256"), digest1).mapErr do (_: auto) -> auto: HashError)
```

Comment:
meh, this is a bit weird - what's `MultiHash` doing here actually except introducing potential errors?

Reply:
Yeah I was kinda surprised too.. there is more (check imports) too.
Not sure tho.. I mean that's another topic/refactor tho maybe.